File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        hashmap = {}
        index = 0
        for val in nums:
            if target - val in hashmap:
                return [index,hashmap[target-val] ]
            else:
                hashmap[val] = index
                
            index += 1
                
        
                

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let hashmap = {};
    let index = 0;
    for (val of nums){
        if(target - val in hashmap){
            return [index, hashmap[target-val] ];
        }
        else{
            hashmap[val] = index;
        }
        index++;
    }
    
};